src/context/index.ts

```diff
@@ -1,27 +1,22 @@
 import { Request } from 'express';
-import { Session, SessionData } from 'express-session';
 import { request } from '../algorithm';
 
 export interface Context {
-  // TODO: keep here until we have login implemented. a partial given it could be undefined
-  session: Session & Partial<SessionData>;
-  // TODO: add context (aka. prisma, auth, etc.)
-  logout: () => Promise<void>;
-  // dependencies
+  req: any;
+  res: any;
```

Comment:
We shouldn't expose these into the context. If you need access you a `user` property define it here instead and populate it within `createContext`